pkg/validations/proxy_validation_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache v2.0 license.
+package validations
+
+import (
+	"testing"
+
+	"github.com/microsoft/moc/pkg/certs"
+)
+
+func Test_ValidateProxyURL(t *testing.T) {
+	caCert, _, err := certs.GenerateClientCertificate("ValidCertificate")
```

Comment:
I assume there are not valid proxy test cases? Success scenario

Reply:
no, there are none

Reply:
Lets call this out in the document and we can resolve this comments. Thanks

Reply:
I added a test case